environments/dev-env/README.md

```diff
@@ -0,0 +1,515 @@
+# KCP-Edge User Kit Environment
+
+## Required Packages:
+   - ko: https://ko.build/install/ 
+   - gcc
+   - docker 
+   - jq
+   - make
+   - go (version expected 1.19)
+   - kind
+   - kubectl  
+
+
+## Supported OS Platforms 
+  - Linux
+  - MacOS
+  - Windows WSL/Ubuntu
+
+
+Run the following script to install the required package (Linux or MacOS ):
+
+```bash
+./install_req.sh
+```
+
+Alternatively, run the following command for MacOS platform:
+
+```bash
+brew install ko gcc jq make go@1.19 kind kubectl
+```
+
+For Windows WSL/Ubuntu platform, follow the instructions [here](docs/wls_ubuntu_platform.md)
+
+##  Contributing
+
+See the [contributing instructions](CONTRIBUTING.md) for detailed instructions on how to deploy and get started with our project.
+
+## Example Scenario
+
+See the [example scenario instructions](docs/example_stages.md) to quickly deploy a kcp-edge demo environment with two sample workloads and two edge clusters. 
+
+## Quickstart
```

Comment:
At this point the reader is wondering "what is the difference between 'example scenario' and 'quickstart'?"